pre_commit/error_handler.py

```diff
@@ -29,6 +31,9 @@ def _log_and_exit(msg, exc, formatted):
         five.to_bytes(msg), b': ',
         five.to_bytes(type(exc).__name__), b': ',
         _to_bytes(exc), b'\n',
+        _to_bytes('pre-commit.version={}\n'.format(pre_commit_version)),
+        _to_bytes('sys.version={}\n'.format(sys.version.replace('\n', ' '))),
+        _to_bytes('sys.executable={}\n'.format(sys.executable)),
```

Comment:
let's put these at the top, perhaps something like this output:

```markdown
    ### version information
    ```
    pre-commit: {...}
    sys.executable: {...}
    sys.version:
        {...}
    ```

    ### error information
    ...
```

Reply:
The format is:
```
### version information
pre-commit.version=...
sys.version=...
sys.executable=...
### error information
An unexpected error has occurred: ...
Check the log at ...
```